### PR TITLE
Bug fixing/45 fix design

### PR DESCRIPTION
**Pull request by martyniukroman**

### CODE REVIEW

**Review by eugent482 — APPROVED** [review]

File: InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.scss [hunk 1]
```diff
@@ -16,34 +36,22 @@ mat-radio-group mat-radio-button {
...
-.inputfile + label * {
-	pointer-events: none;
-}
-
 .image-view {
     margin: 15px;
-    border-radius: 2%;
-    width: 250px;
+    //xwidth:300px;
+    max-width:350px;
```
Comment (voznovych) [inline InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.scss:50]: Space after "max-width:".

File: InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.ts [hunk 2]
```diff
@@ -45,18 +45,21 @@ export class SignUpComponent implements OnInit {
...
               this.imageUrl = event.target.result;
-              this.isImageValid = true;  
+              this.isImageValid = true; 
+              // alert("this.service.form.valid: " + this.service.form.valid);
+              // alert("isImageValid: " + this.isImageValid);
             }
             else
             {
-                this.service.form.invalid;
+                this.service.form.invalid; // what is this line?
```
Comment (voznovych) [inline InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.ts]: Remove this comment.
P.s.
This makes model invalid and show mat-errors.

File: InternetHospitalWebUI/src/app/Services/registration.service.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ export class RegistrationService {
     Email: new FormControl('', Validators.email),
     Password: new FormControl('', Validators.required),
     ConfirmPassword: new FormControl('', [Validators.required, compareValidator('Password')]),
-    Role: new FormControl('', Validators.required),
+    Role: new FormControl(''),
```
Comment (voznovych) [inline InternetHospitalWebUI/src/app/Services/registration.service.ts:21]: Why role isn't required? If i'm hatsker, i can remove selected radio by devtools and send form without it.

**Review by SergiyKozachok — APPROVED** [review]

File: InternetHospitalWebUI/src/app/Services/registration.service.ts (hunk 3 above)
Comment (martyniukroman) [reply InternetHospitalWebUI/src/app/Services/registration.service.ts:21]: The role anyway is validated on back-end
Now we have the default checked patient radiobutton
so what the problem?

File: InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.scss (hunk 1 above)
Comment (martyniukroman) [reply InternetHospitalWebUI/src/app/Components/Authorization/sign-up/sign-up.component.scss:50]: removed

**Review by voznovych — APPROVED** [review]

**Pull request merged**